sources/s5b/sections/01-basic_subjects.adoc

```diff
@@ -8,7 +8,8 @@
 |===
 | Topic/Element | Content | Learning outcomes
 
-| B1.1 Linear Algebra _(B)_
+| B1.1 Linear Algebra +
+_(B)_
```

Comment:
@anermina let's not encode these non-semantic elements. For Metanorma, it is not important whether the `_(B)_` occurs on the same line or not. Thanks.

Reply:
Got it! Changes are applied in both IHO S5a and IHO S5b.